docs/_templates/search-field.html

```diff
@@ -0,0 +1,18 @@
+<form class="bd-search d-flex align-items-center" action="{{ pathto('search') }}" method="get">
+  <div class="input-group">
+    <label for="q" class="sr-only">Search string
+    </label>
+    <div class="input-group-prepend">
+      <span class="input-group-text" id="shortcut">
+        <kbd id="search-shortcut">⌘</kbd>&nbsp;<kbd>K</kbd>
+      </span>
+    </div>
```

Comment:
The outline is nicer with the dedicated bootstrap class.
```suggestion
    <div class="input-group-append">
      <span class="input-group-text" id="shortcut">
        <kbd id="search-shortcut">⌘</kbd>&nbsp;<kbd>K</kbd>
      </span>
    </div>
```
and move after the input field (and the to be added search icon)